bindings/wasm/examples-account/src/node.ts

```diff
@@ -36,6 +37,8 @@ async function main() {
             return await unchecked();
         case "multiple_identities":
             return await multipleIdentities();
+        case "key_exchange":
+            return await keyExchange();
```

Comment:
Should match the Rust account example's name. I think "encryption" is better than "key_exchange".